tests/src/container-smoke.spec.ts

```diff
@@ -27,12 +27,14 @@ import { waitUntil, waitWhile } from './utility/wait';
 import { deleteContainer, deleteImage } from './utility/operations';
 import { ContainerState } from './model/core/states';
 import type { ImagesPage } from './model/pages/images-page';
+import { ContainersPage } from './model/pages/containers-page';
 
 let pdRunner: PodmanDesktopRunner;
 let page: Page;
 const imageToPull = 'ghcr.io/linuxcontainers/alpine';
 const imageTag = 'latest';
 const containerToRun = 'alpine-container';
+const containerList = ['first', 'second', 'third'];
```

Comment:
Are the names in this list arbitrary? Where does the initialization of said containers happen?

Reply:
ah, I see, thank you for the explanation